action.yml

```diff
@@ -73,4 +73,4 @@ runs:
             --field body=@- \
             --field tag_name="$( gh api "$RELEASE_URL" --jq .tag_name 2>&19 )" |
           jq .body | # escaped, but with quotes
-          xargs -I% echo ::set-output name=body::%
+          xargs -I% echo "body=%" >> "$GITHUB_OUTPUT"
```

Comment:
`jq` isn't being called with `--raw-output`, so it already wrapped its output with quotes. Quoting it (as `%`) again will remove them. So, don't worry, and don't quote `%` in this case.

OR add `--raw-output` and leave the quotes (and worry about nested quotes)